controllers/grid/users/author/AuthorGridHandler.inc.php

```diff
@@ -308,8 +308,14 @@ public function canAdminister($user)
      */
     protected function loadData($request, $filter = null)
     {
-        $authorDao = DAORegistry::getDAO('AuthorDAO'); /** @var AuthorDAO $authorDao */
-        return $authorDao->getByPublicationId($this->getPublication()->getId(), true, false);
+        $authors = Repo::author()->getMany(
+            Repo::author()
+                ->getCollector()
+                ->filterByPublicationIds([$this->getPublication()->getId()])
+                ->orderBy(Repo::author()->getCollector()::ORDERBY_ID)
```

Comment:
I think it in the author grid we want to return authors ordered by sequence, not by ID. That way when the user finishes ordering them, they see the order that they put them in.

Reply:
Changed